### PR TITLE
docs: manual of container labels and annotaions under cri

### PR DESCRIPTION
**Pull request by zhuangqh**
Signed-off-by: zhuangqh <zhuangqhc@gmail.com>

### Ⅰ. Describe what this PR did

- add missing document for vm pass through config annotation
- explain the container label rule under CRI implement

### CODE REVIEW

File: docs/kubernetes/pouch_cri_annotations_supported.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 * [Overview](#overview "Overview")
 * [The Annotations Supported](#the-annotations-supported "The Annotations Supported")
-  * [Make runtime choosing supported](#make-runtime-choosing-supported "Make runtime choosing supported")
-  * [Make lxcfs configurable supported](#make-lxcfs-configurable-supported "Make lxcfs configurable supported")
-* [Pull Request](#pull-request "Pull Request")
+  * [Runtime choosing](#runtime-choosing "Runtime choosing")
```
Comment (allencloud) [inline docs/kubernetes/pouch_cri_annotations_supported.md]: Can we use RuntimeClass in Kubernetes to support runv/Kata which is described in https://github.com/alibaba/pouch/issues/2314? @Starnop @zhuangqh

Comment (zhuangqh) [reply docs/kubernetes/pouch_cri_annotations_supported.md]: We use annotations to support runtime choosing for legacy kubernetes. RuntimeHandler of kubernetes 1.12 is also supported

Comment (allencloud) [reply docs/kubernetes/pouch_cri_annotations_supported.md]: If that, I think we should add detailed instructions in this document. 
< 1.12, suggest using annotation;
>= 1.12, use runtime class

In addition, we should have a plan whether to deprecate the annotation for future.

**Review by cxz66666 — APPROVED** [review]

**Review by cxz66666 — CHANGES_REQUESTED** [review]
Hope you can fix these questions

**Comment by cxz66666** [comment]
Thank you for your contribution! because you don't response for a long time, I will close this PR and reopen [another one](https://github.com/alibaba/pouch/pull/3035). ❤️

**Pull request closed without merging**